DATA/crustmap/readme.txt

```diff
@@ -28,7 +28,7 @@ density    crustmaps (e.g. eucrustr3.cmap - eucrustr7.cmap)
 
 these are simple ascii data files and you can change them easily. The current crustmap resolution
 is set to 1/4 degree.
-The Values are ordered on a grid that runs from 90° lat (0° lng to 180° lng ) to -90° lat.
+The Values are ordered on a grid that runs from 90? lat (0? lng to 180? lng ) to -90? lat.
```

Comment:
Should write out degrees here, or just remove the `?` since it looks like you aren't sure whether it's 90.